findoutlie/metrics.py

```diff
@@ -22,11 +19,17 @@ def dvars(img):
         volumes in `img`.
     """
 
-    # Hint: remember 'axis='.  For example:
-    # In [2]: arr = np.array([[2, 3, 4], [5, 6, 7]])
-    # In [3]: np.mean(arr, axis=1)
-    # Out[2]: array([3., 6.])
-    #
-    # You may be be able to solve this in four lines, without a loop.
-    # But solve it any way you can.
+    # Ensure the image is loaded and has the necessary data.
+    if img is None:
```

Comment:
Generally you will find Python code doesn't do many specific checks like these.  Typically you'd let the error occur on line 27 when you try to do `get_fdata` on `None`.  The reason for not doing the checks is that they make the code longer and less readable.

Reply:
Thanks for your review, I will change my code using try block to make it more readable and make a new pull request.  :)